plugins/fishlim/fish.h

```diff
@@ -29,10 +30,12 @@
 
 #include <glib.h>
 
-char *fish_encrypt(const char *key, size_t keylen, const char *message);
-char *fish_decrypt(const char *key, size_t keylen, const char *data);
-char *fish_encrypt_for_nick(const char *nick, const char *data);
-char *fish_decrypt_from_nick(const char *nick, const char *data);
+enum fish_mode {ECB = 0x1, CBC = 0x2};
```

Comment:
Just because its an enum doesn't mean you have to have a short name, the previous one was more clear.

Also format one per line.

Reply:
Yes, I'm agree